lib/active_model/serializer.rb

```diff
@@ -105,7 +105,7 @@ def self.root_name
       name.demodulize.underscore.sub(/_serializer$/, '') if name
     end
 
-    attr_accessor :object, :root, :meta, :meta_key, :scope
+    attr_accessor :object, :root, :meta, :meta_key, :links, :scope
```

Comment:
Well, links is a top-level member of the json-api response object.  Pagination is a usage of the links object.  So, my thinking is that this PR adds 'links' and the pagination one uses it to add pagination data. Make sense?